src/corehost/cli/nethost/nethost.cpp

```diff
@@ -18,6 +18,9 @@ namespace
     }
 }
 
+#if defined(_WIN32) && defined(_TARGET_X86_)
+    #pragma comment(linker, "/export:get_hostfxr_path=_get_hostfxr_path@12")
```

Comment:
Wouldn't it be easier to use in `C` if the export did not include the `@12`?

Reply:
The `_get_hostfxr_path@12` export is the one that already exists (based `extern C` and `declspec(dllexport)`). This is adding the `get_hostfxr_path` export which points to `_get_hostfxr_path@12`.

Reply:
I have no idea how this is supposed to work... but is there maybe a way to only have one export, the unmangled one?

Reply:
Using the `#pragma comment`, when I removed the decorated one, it broke going through import lib. I switched it to using a .def for exports (only exporting the unmangled one) which worked.

Honestly, I'm unclear why (the exports and symbols are exactly the same from what I can tell in both cases), but I guess this is more consistent with how other libhosts using `__stdcall` unmangle their exports anyways...